### PR TITLE
Update supported kubernetes versions

### PR DESCRIPTION
**Pull request by mrIncompetent**
**What this PR does / why we need it**:
Update supported kubernetes versions & drop kubernetes v1.8.

**Release note**:
```release-note
Support for Kubernetes v1.8 has been dropped. The control planes of all clusters running 1.8 will be automatically updated
```

### CODE REVIEW

**Comment by alvaroaleman** [comment]
Maybe reword the second sentence in the release note to "The control planes of all clusters running 1.8 will be automatically updated"?

**Comment by mrIncompetent** [comment]
@alvaroaleman done

**Review by alvaroaleman — APPROVED** [review]
